packages/atlas/src/providers/wallet/tmpwallet/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './wallet-connect.js'
+export * from './types.js'
+export * from './consts.js'
```

Comment:
Why are these `.js` files ?

Reply:
these are used as these files are copied from the actual lib I'm currently working on (paused atm so I'm not sure if I need to finalise it soon). Therefore, I left them untouched